docs/README.md

```diff
@@ -1,56 +1,102 @@
 # What is cmdalias
 
-`cmdalias` is a tool that I created for myself because I was tired of using too many keystrokes for commands that i'm using everyday in my shell. For example I often want to list images from my docker.
+`cmdalias` is an alias tool inspired from git-config that I created for myself. Moreover, it lets you create
+nested aliases.
```

Comment:
It would be nice to say something like *context aware nested / subalias* and add examples in the second part of the doc. 
Not sure which example to choose but if you have a `cmdalias` with
```
d = docker {
i = images;
};
```

You can type `d i` for `docker images` but `d run i` will run `d i i` will run `docker images i`. 

And it can be done at any depth.